lib/main.js

```diff
@@ -20,6 +20,7 @@ import * as utils from './utils.js';
  * @property {string} [notFoundCacheControl="public, max-age=5"]
  * @property {number} [pkgMaxFileSize=10000000] The limit in bytes before PUT /pkg/ starts returning 413 Content Too Large
  * @property {number} [mapMaxFileSize=1000000] The limit in bytes before PUT /map/ starts returning 413 Content Too Large
+ * @property {number} [imgMaxFileSize=10000000] The limit in bytes before PUT /img/ starts returning 413 Content Too Large
```

Comment:
Do we maybe want to set the default here higher, like `20_000_000`?

Reply:
err, I just realised, we don't actually set default values, we just say we do in the types...